resources/core/charts/event-bus/charts/publish/templates/deployment.yaml

```diff
@@ -30,7 +30,7 @@ spec:
             - --client_id=$(POD_NAME)
             - --max_requests={{ .Values.global.publish.maxRequests }}
             - --nats_streaming_cluster_id={{ .Values.global.natsStreaming.clusterID }}
-            - --nats_url=nats://{{ template "nats-streaming.fullname" . }}:{{ toString .Values.global.natsStreaming.ports.client }}
+            - --nats_url=nats://{{ .Values.global.natsStreaming.fullname }}.{{ .Values.global.natsStreaming.namespace }}:{{ toString .Values.global.natsStreaming.ports.client }}
```

Comment:
I understand your point, however, first of all, it's a convention (at least in the event-bus charts) plus somehow it's kinda more explicit because as you can see or example, adding the namespace explicitly gives insight about how the service now is discoverable over the network (`nats-streaming.natss` instead of `natsstreaming`)

Reply:
But in the same time it makes configuration less elastic. Having `url` here would even allow to use external `natss`. Also If you make it only `url` Whole `global.natsStreaming` becomes just one parameter, because you're not using those parameters to anything else then building URL. In my opinion it is leaking logic: core shouldn't care about where the natss come from.

Reply:
I got your point, OK, will change it